default/decorators.py

```diff
@@ -14,11 +19,35 @@ def wrap(request, *args, **kwargs):
     return wrap
 
 
-def require_files(function):
+def validate_files(function):
```

Comment:
I think it would be best to separate this into another decorator, with a clearer name like `extract_last_result` or something similar.

Reply:
Done.